test/unit/url-mapper-spec.js

```diff
@@ -152,11 +106,43 @@ describe('url mapper', function() {
       check(specific.url, specific);
     });
 
+    it('matches, if no trailing slash', function() {
+      const noTrailing = {
+        url: 'http://foo.com',
+        newUrl: 'newUrl'
+      };
+
+      set(noTrailing);
+      check('http://foo.com', noTrailing);
+      check('http://foo.com/', noTrailing);
+    });
+
+    it('matches, if trailing slash', function() {
+      const trailingSlashes = {
+        url: 'http://foo.com/',
+        newUrl: 'newUrl'
+      };
+
+      set(trailingSlashes);
+      check('http://foo.com', trailingSlashes);
+      check('http://foo.com/', trailingSlashes);
+    });
+
     it('matches wildcards', function() {
       set(oneWildcard);
       check('http://foo.com/1/baz', oneWildcard);
     });
 
+    it('doesn\'t match wildcard regardless of trailing slash or not', function() {
```

Comment:
Ah, I think I understand. These are acting more as integration tests, checking both in-and-out, rather than just a single function on `url-mapper`?